composer.json

```diff
@@ -29,13 +29,19 @@
     "autoload": {
         "psr-4": { "Omnipay\\Mollie\\" : "src/" }
     },
+    "autoload-dev": {
+        "psr-4": { "Omnipay\\Mollie\\Test\\": "tests/" }
+    },
     "require": {
         "omnipay/common": "^3"
     },
     "require-dev": {
         "omnipay/tests": "^3",
         "squizlabs/php_codesniffer": "^3",
-        "phpro/grumphp": "^0.14"
+        "phpro/grumphp": "^0.14",
+        "phpmd/phpmd": "@stable",
```

Comment:
Can you provide versions for this? Or is there a specific reason to not require a version?